src/firewall/core/io/policy.py

```diff
@@ -77,6 +77,16 @@ def common_startElement(obj, name, attrs):
             else:
                 log.warning("Protocol '%s' already set, ignoring.",
                             attrs["value"])
+    
+    elif name == "tcp-mss-clamp":
+        if obj._rule:
+            if obj._rule.element:            
+                log.warning("Invalid rule: More than one element in rule '%s', ignoring.",
+                            str(obj._rule))
+                obj._rule_error = True
+                return True
+            obj._rule.element = rich.Rich_Tcp_Mss_Clamp(attrs["value"])
+    
```

Comment:
Would it be best to leave a FirewallError rather than a log.warning? Because looking at similar code, with a similar errors it says something like log.warning("Invalid rule: icmp-block '%s' outside of rule",attrs["name"]) rather than raising FirewallError.

Reply:
Use `log.warning()`... that way the XML parsing can continue. Seems like the only place we raise exceptions in this area is inside the check functions.